handlers/admin/adminPageSizePage.go

```diff
@@ -49,5 +49,5 @@ func AdminPageSizePage(w http.ResponseWriter, r *http.Request) {
 		Max:     cd.Config.PageSizeMax,
 		Default: cd.Config.PageSizeDefault,
 	}
-	handlers.TemplateHandler(w, r, "admin/pageSizePage.gohtml", data)
+	handlers.TemplateHandler(w, r, "pageSizePage.gohtml", data)
```

Comment:
@jules, this page must be in `admin/` as it's admin move the page rather than change this if required.